pulp_2to3_migration/app/pre_migration.py

```diff
@@ -81,8 +85,12 @@
         type=content_type,
         timestamp=last_updated))
 
-    # query only newly created/updated items
-    mongo_content_qs = content_model.pulp2.objects(_last_updated__gte=last_updated)
+    if premigrate_hook:
+        mongo_content_qs = content_model.pulp2.objects(
+            _last_updated__gte=last_updated, id__in=premigrate_hook)
```

Comment:
Following the idea of having only function reference, I'd call it here and the variable names might be clearer as well:
```
if premigrate_hook:
    pulp2_content_ids = premigrate_hook()
    mongo_content_qs = content_model.pulp2.objects(
         _last_updated__gte=last_updated, id__in=pulp2_content_ids)
```